src/seeders/20200212121323-User.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
     'Users',
     [
       {
-        id: uuid(),
+        id: "d0551d02-46e6-4b79-9c3b-8d378c178243",
```

Comment:
why this change?

Reply:
It Baraka who put the uuid().... which might mean I didn't rebase well.

Reply:
Oh, that should be it. It seemed as though you'd undone Baraka's changes